x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/add_domain/validation_state_icon.tsx

```diff
@@ -14,7 +14,14 @@ import { CrawlerDomainValidationStepState } from '../../types';
 export const ValidationStateIcon: React.FC<{ state: CrawlerDomainValidationStepState }> = ({
   state,
 }) => {
-  if (state === 'valid') return <EuiIcon color="success" type="checkInCircleFilled" />;
-  if (state === 'invalid') return <EuiIcon color="danger" type="crossInACircleFilled" />;
-  return <EuiLoadingSpinner />;
+  switch (state) {
+    case 'valid':
+      return <EuiIcon color="success" type="check" />;
```

Comment:
I guess the type is changed here because of the visual change we also made in the standalone UI, correct?

Reply:
Correct, I'm aligning it with the changes done in the standalone UI PR